public/app/features/alerting/unified/state/actions.ts

```diff
@@ -884,7 +895,7 @@ export const updateRulesOrder = createAsyncThunk(
             rules: newRules,
           };
 
-          await setRulerRuleGroup(rulerConfig, namespaceName, payload);
+          await setRulerRuleGroup(rulerConfig, folderUid, payload);
```

Comment:
Should this be `namespaceName ?? folderUid` to make it work for both cloud and GMA rules?